docs/getting-started-guides/scratch.md

```diff
@@ -642,49 +624,57 @@ If you are using the HTTPS approach, then set:
 }
 ```
 
-The `/etc/ssl` mount allows the apiserver to find the SSL root certs so it can
-authenticate external services, such as a cloud provider.
-
-The `/srv/kubernetes` mount allows the apiserver to read certs and credentials stored on the
-node disk.
-
-Optionally, you may want to mount `/var/log` as well and redirect output there.
-
-#### Starting Apiserver
-
-Place the completed pod template into the kubelet config dir
-(whatever `--config=` argument of kubelet is set to, typically
-`/etc/kubernetes/manifests`).
-
-Next, verify that kubelet has started a container for the apiserver:
-
-```console
-$ sudo docker ps | grep apiserver:
-5783290746d5        gcr.io/google_containers/kube-apiserver:e36bf367342b5a80d7467fd7611ad873            "/bin/sh -c '/usr/lo'"    10 seconds ago      Up 9 seconds                              k8s_kube-apiserver.feb145e7_kube-apiserver-kubernetes-master_default_eaebc600cf80dae59902b44225f2fc0a_225a4695
-
-```
+Here are some apiserver flags you may need to set:
+  - `--cloud-provider=` see [cloud providers](#cloud-providers)
+  - `--cloud-config=` see [cloud providers](#cloud-providers)
+  - `--address=${MASTER_IP}`.
+   - or `--bind-address=127.0.0.1` and `--address=127.0.0.1` if you want to run a proxy on the master node.
```

Comment:
This is old/misleading; should mention & explain:
--insecure-bind-address
--bind-address
--advertise-address